doc/en/components/grids/_shared/toolbar.md

```diff
@@ -0,0 +1,669 @@
+---
+title: {Platform} {Component} {ComponentTitle} - {ProductName} for {Platform}
+_description: Use {Platform} {Component} {ComponentTitle} for essential UI operations. Hosts different UI controls for the Grid’s features - column hiding, pinning, excel exporting, etc.
+_keywords: {Platform}, {ComponentTitle}, {ComponentTitle}, {ProductName}, Infragistics
+mentionedTypes: [{ComponentApiMembers}]
+sharedComponents: ["Grid", "TreeGrid", "HierarchicalGrid"]
+---
+
+# {Platform} {ComponentTitle} Toolbar
+
+The {ComponentTitle} in {ProductName} for {Platform} provides an [`GridToolbarComponent`]({environment:{Platform}ApiUrl}/classes/gridtoolbarcomponent.html) which is essentially a container for **UI** operations. The {Platform} toolbar is located at the top of the {Platform} component, i.e the {ComponentTitle} and it matches its horizontal size. The toolbar container can host predefined UI controls for the following {ComponentTitle}'s features:
+
+ - Column Hiding
+ - Column Pinning
+ - Excel Exporting
+ - Advanced Filtering
+
+or just any other custom content. The toolbar and the predefined UI components support {Platform} events and expose API for developers.
+
+## {Platform} Toolbar Grid Example
+
+@@if (igxName === 'IgxGrid') {
+
+<code-view style="height:420px" 
+           data-demos-base-url="{environment:demosBaseUrl}" 
+           iframe-src="{environment:demosBaseUrl}/grid/grid-toolbar-sample-1" alt="{Platform} Toolbar Grid Example">
+</code-view>
+
+}
+@@if (igxName === 'IgxTreeGrid') {
+
+<code-view style="height:420px" 
+           data-demos-base-url="{environment:demosBaseUrl}" 
+           iframe-src="{environment:demosBaseUrl}/tree-grid/treegrid-toolbar-4" alt="{Platform} Toolbar Grid Example">
+</code-view>
+
+}
+@@if (igxName === 'IgxHierarchicalGrid') {
+
+<code-view style="height:510px" 
+           data-demos-base-url="{environment:demosBaseUrl}" 
+           iframe-src="{environment:demosBaseUrl}/hierarchical-grid/hierarchical-grid-toolbar-title" alt="{Platform} Toolbar Grid Example">
+</code-view>
+
+}
```

Comment:
replace above 3 code-view blocks with just one code-view and remove component build flag because routing path to the sample will be the same for all grid samples in Sample Browser.